.github/workflows/e2e.yaml

```diff
@@ -14,8 +14,8 @@ jobs:
       fail-fast: false
       matrix:
         kubernetes:
-          - v1.30.10
-          - v1.33.0
+          - v1.31.9
+          - v1.33.1
```

Comment:
Point this out (not blocking): we may want to update the test matrix to keep the min k8s/Tekton and latest k8s/Tekton versions in sync. See https://github.com/shipwright-io/build/blob/main/.github/workflows/ci.yml#L79-L85